compiler/crates/relay-compiler/src/main.rs

```diff
@@ -0,0 +1,56 @@
+use env_logger::Env;
+use log::{error, info};
+use relay_compiler::{compiler::Compiler, config::Config};
+use std::{path::PathBuf, sync::Arc};
+use structopt::StructOpt;
+
+#[derive(StructOpt)]
+#[structopt(
+    name = "Relay Compiler",
+    about = "Compiler to produce Relay generated files."
+)]
+struct Opt {
+    /// Compile and watch for changes
+    #[structopt(long, short)]
+    watch: bool,
+
+    /// Path to the compiler config file
+    config: PathBuf,
+
+    /// Root directory of the project
+    root: PathBuf,
```

Comment:
I think the `root` should actually come from the `config.json` file. Maybe with the directory of the config.json the default and otherwise a relative root configured in the config.json. Thoughts?

Reply:
Yup, that makes sense to me 👍